deeprank/operate/pdb.py

```diff
@@ -0,0 +1,110 @@
+import numpy
+import logging
+
+from deeprank.models.pair import Pair
+from deeprank.models.atom import Atom
+from deeprank.models.residue import Residue
+
+
+_log = logging.getLogger(__name__)
+
+def get_squared_distance(position1, position2):
```

Comment:
Can we combine get_squared_distance and get_distance?
Just to reduce some lines of the code

Reply:
I removed the squared distance function and the code is now using the normal distance function.